Content.Server/Power/Generation/Teg/TegSensorData.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Content.Server.Power.Components;
+
+namespace Content.Server.Power.Generation.Teg;
+
+/// <summary>
+/// Sensor data reported by the <see cref="TegGeneratorComponent"/> when queried over the device network.
+/// </summary>
+/// <seealso cref="TegSystem"/>
+public sealed class TegSensorData
```

Comment:
y class

Reply:
What else would it be?

Reply:
struct from rust

Reply:
A struct wouldn't make sense, the value is always boxed.